addons/account/models/partner.py

```diff
@@ -113,9 +114,15 @@ def write(self, vals):
         if zip_from or zip_to:
             for rec in self:
                 vals['zip_from'], vals['zip_to'] = self._convert_zip_values(zip_from or rec.zip_from, zip_to or rec.zip_to)
+        self.clear_caches()
```

Comment:
It's probably better not to do that all the time. 
If we are not changing the region selection fields of the fiscal position (country, state, zip range) or auto_apply is  False, we don't need to clear the cache, right ?

Reply:
Indeed, but from what I have been told, this data only changes once a year, and therefore it does not really have an impact. I can indeed filter for cleaning the cache.

Reply:
It changes when someone reconfigures something. So, indeed, not the most common operation, but I guess we should gain what we can :)

Reply:
At the expense of readability for one invalidation per year? I looked around a bit, but it makes for a weird condition and if someone changes relationships or else they will have to change the condition (without necessarily seeing the error as it is only when when there is a specific change that must invalidate the cache)